woocommerce/woocommerce-ios
# Bump deployment target to iOS 14.0

As discussed in p91TBi-51t-p2, we are bumping the required iOS version to 14.0.

This will happen in the 6.8 release, set to be released June 14.

## Changes

- [x] Update all the targets to iOS 14.0
- [x] `SKStoreReviewController.requestReview()` was deprecated in iOS 14

## To test

I'd recommend general testing of the app.
The only thing that has changed besides the target is the app review prompts, so maybe do some extra testing there. I had to modify the code to make sure I got the prompt and make sure `AppRatingManager.shouldPromptForAppReview(section:)` returned `true`

Update release notes:
